prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -160,38 +166,59 @@ data class Hl7Configuration
     override val format: Report.Format get() = if (useBatchHeaders) Report.Format.HL7_BATCH else Report.Format.HL7
 
     @get:JsonIgnore
-    override val defaults: Map<String, String> get() {
-        val receivingApplication = when {
-            receivingApplicationName != null && receivingApplicationOID != null ->
-                "$receivingApplicationName^$receivingApplicationOID^ISO"
-            receivingApplicationName != null && receivingApplicationOID == null ->
-                receivingApplicationName
-            else -> ""
-        }
-        val receivingFacility = when {
-            receivingFacilityName != null && receivingFacilityOID != null ->
-                "$receivingFacilityName^$receivingFacilityOID^ISO"
-            receivingFacilityName != null && receivingFacilityOID == null ->
-                receivingFacilityName
-            else -> ""
-        }
-        val reportingFacility = when {
-            reportingFacilityName != null && reportingFacilityId != null && reportingFacilityIdType == null ->
-                "$reportingFacilityName^$reportingFacilityId^CLIA"
-            reportingFacilityName != null && reportingFacilityId != null && reportingFacilityIdType != null ->
-                "$reportingFacilityName^$reportingFacilityId^$reportingFacilityIdType"
-            reportingFacilityName != null && reportingFacilityId == null ->
-                reportingFacilityName
-            else -> ""
+    override val useBatching: Boolean get() = useBatchHeaders
+
```

Comment:
I think when that has come up in the past, we have said that we're not expecting to need to support that, but if we did, this framework allows for that decently well; we would probably just introduce a `useBatching` json property for hl7 configs and then use that property instead of overriding the getter to return `useBatchHeaders` like we're doing here. Then in the code, we'd just have to handle `useBatching` and `useBatchHeaders` separately. So I don't think we're boxing ourselves in here, and we can handle that separation in it's own ticket if/when that requirement shows up.

Reply:
Something else I just thought about is if all FHIR receivers would want to receive batch FHIR in ndjson or if we would need to support another format like sending the bundles in a JSON array. But we can probably worry about that later. Do we know what's needed for the connectathon?